assets/gofish/lang/zh_tw.json

```
@@ -0,0 +1,132 @@
+  "item.gofish.endfish_and_chorus.tooltip_2": "讓你想起了炸魚薯條",
+  "item.gofish.baked_endfish": "烤終界魚",
+  "item.gofish.baked_endfish.tooltip_1": "烤終界魚肉質軟嫩",
+  "item.gofish.baked_endfish.tooltip_2": "入口即化",
+  "item.gofish.grilled_blackstone_trout": "烤黑石鱒魚",
+  "item.gofish.grilled_blackstone_trout.tooltip_1": "小火慢烤製成",
+  "item.gofish.grilled_blackstone_trout.tooltip_2": "美味且飽足感十足",
+  "item.gofish.grilled_blackstone_deluxe": "黑石鱒魚大餐",
+  "item.gofish.grilled_blackstone_deluxe.tooltip_1": "烤黑石鱒魚搭配抹了奶油的馬鈴薯",
+  "item.gofish.grilled_blackstone_deluxe.tooltip_2": "再加上一點新鮮的綠葉蔬菜點綴",
+  "item.gofish.grilled_blackstone_deluxe.tooltip_3": "一道美味的大餐就完成了",
+
+  "item.gofish.blaze_rod": "烈焰桿釣竿",
+  "item.gofish.skeletal_rod": "骨製釣竿",
+  "item.gofish.diamond_reinforced_rod": "強化鑽石釣竿",
+  "item.gofish.ender_rod": "終界釣竿",
+  "item.gofish.frosted_rod": "冷凍釣竿",
+  "item.gofish.slime_rod": "史萊姆釣竿",
+  "item.gofish.soul_rod": "靈魂釣竿",
+  "item.gofish.matrix_rod": "星晶釣竿",
+
+  "item.gofish.celestial_rod": "天體釣竿",
+  "item.gofish.celestial_rod.tooltip_1": "晚上時海洋的祝福 +1",
+
+  "block.gofish.wooden_crate": "木箱",
+  "block.gofish.iron_crate": "鐵箱",
+  "block.gofish.golden_crate": "金箱",
+  "block.gofish.diamond_crate": "鑽石箱",
+  "block.gofish.soul_crate": "靈魂箱",
+  "block.gofish.frosted_crate": "冷凍箱",
+  "block.gofish.gilded_blackstone_crate": "鑲金黑石箱",
+  "block.gofish.slimey_crate": "史萊姆箱",
+  "block.gofish.fiery_crate": "火熱的箱子",
+  "block.gofish.supply_crate": "補給箱",
+  "block.gofish.astral_crate": "太空箱",
+  "block.gofish.end_crate": "終界箱",
+
+  "advancements.fishing.root.title": "釣魚之旅",
+  "advancements.fishing.root.description": "想釣魚？一根釣竿足矣。",
+  "advancements.fishing.full_moon_fish.title": "在滿月下釣魚",
+  "advancements.fishing.full_moon_fish.description": "在滿月時捕獲所有特殊種類的魚",
+  "advancements.fishing.weather_fish.title": "雷雨交加的海洋",
+  "advancements.fishing.weather_fish.description": "捕獲所有天氣系列的魚",
+  "advancements.fishing.golden_lure.title": "黃金大魚餌",
+  "advancements.fishing.golden_lure.description": "在釣魚途中發現一條黃金魚，拿來當魚餌感覺不錯。",
+  "advancements.fishing.seaweed.title": "上好海藻",
+  "advancements.fishing.seaweed.description": "在釣魚途中撈上來一些海藻，這應該會是一株大的。",
+  "advancements.fishing.seaweed_eel.title": "間諜海藻",
+  "advancements.fishing.seaweed_eel.description": "捕獲一隻海藻鰻",
+  "advancements.fishing.baked_seaweed.title": "海藻升級",
+  "advancements.fishing.baked_seaweed.description": "把沒人要的海藻變成可口的烤海藻",
+  "advancements.fishing.grilled_blackstone_deluxe.title": "新鮮又美味",
+  "advancements.fishing.grilled_blackstone_deluxe.description": "用烤黑石鱒魚加上烤海藻煮一道令人垂涎三尺的料理",
+  "advancements.fishing.aquatic_astral_stew.title": "仰望星空",
+  "advancements.fishing.aquatic_astral_stew.description": "用你在滿月時抓到的魚做一道海洋太空燉湯",
+  "advancements.fishing.astral_crate.title": "天空之外的箱子",
+  "advancements.fishing.astral_crate.description": "在滿月時釣到一個太空箱",
+  "advancements.fishing.wooden_crate.title": "簡易箱子",
+  "advancements.fishing.wooden_crate.description": "釣到一個木盒，對著空氣 Shift + 右鍵來打開。",

```

```suggestion
  "advancements.fishing.wooden_crate.description": "釣到一個木盒，對著空氣 Shift + 點擊右鍵開啟。",
```
為了翻譯一致性，``Open`` 習慣上是譯作 ``開啟``